push_notification_ui/static/js/main.js

```diff
@@ -1,12 +1,31 @@
-import {getServerPublicKey, getBaseUrl, logError} from "./utils.js"
-import {registerSubscriber} from "./requests.js"
 import {grantingNotificationPermission} from "./register-subscriber.js"
+import {createNotification, sendNotification} from "./requests.js"
+import {getNotificationItems} from "./notifications.js"
 
-var subscribe_notification_btn = document.getElementById("subscriber_notification_btn")
+const subscribe_notification_btn = document.getElementById("subscriber_notification_btn")
+const notification_form = document.getElementById('notification-form')
+const reload_notification = document.getElementById('refresh-notifications-btn')
+const notification_items = document.getElementById('notifications-items')
+const send_notification_btn = document.getElementById('send-notification-btn')
+
+getNotificationItems(notification_items)
 
 subscribe_notification_btn.addEventListener('click', function(event) {
     event.preventDefault()
-    var permission = grantingNotificationPermission()
-    console.log(permission)
+    grantingNotificationPermission()
+});
+
+notification_form.addEventListener('submit', function(event) {
+    event.preventDefault()
+    createNotification(notification_form)
 });
 
+reload_notification.addEventListener('click', function(event) {
+    event.preventDefault()
+    getNotificationItems(notification_items)
+})
+
+send_notification_btn.addEventListener('click', function(event) {
+    event.preventDefault()
```

Comment:
Why do we want to `preventDefault` of `click` events?

Reply:
Noted, removing default action on click events